src/client/utils.js

```diff
@@ -62,31 +62,18 @@ function pollResult (txID, resolve, reject, nodeClient, lookupRetries, lookupTim
 */
 export function TransactionForResult (nodeClient, action, lookupRetries = 5, lookupTimeout = 500) {
   return new Promise((resolve, reject) => {
-    // Get nonce.
-    nodeClient.nonceLookup().then(result => {
+    action.nonce = 3
```

Comment:
This should be removed.  Why would you override the nonce of the action that is passed to this function?

Reply:
Fixed. Thanks.